config.json

```diff
@@ -470,5 +484,9 @@
         "mathematics"
       ]
     }
-  ]
+  ],
+  "foregone": [
```

Comment:
I believe all the additions here are now at the top of the file, so you don't need any of this group of changes

Reply:
Yep, this is now fixed. Thanks for the hint.